test/imported.test.js

```diff
@@ -9,13 +9,70 @@ describe('cloudant imported features', function() {
   before(function() {
     IMPORTED_TEST = true;
   });
+
   after(function() {
     IMPORTED_TEST = false;
   });
-  require('loopback-datasource-juggler/test/datatype.test.js');
-  require('loopback-datasource-juggler/test/basic-querying.test.js');
-  require('loopback-datasource-juggler/test/hooks.test.js');
-  require('loopback-datasource-juggler/test/relations.test.js');
+
   require('loopback-datasource-juggler/test/include.test.js');
-  require('loopback-datasource-juggler/test/manipulation.test.js');
+  require('loopback-datasource-juggler/test/common.batch.js');
```

Comment:
I assume `common.batch.js` file in Juggler has the following test files included?
```
-  require('loopback-datasource-juggler/test/datatype.test.js');		 
-  require('loopback-datasource-juggler/test/basic-querying.test.js');		
-  require('loopback-datasource-juggler/test/hooks.test.js');		
-  require('loopback-datasource-juggler/test/relations.test.js');
-  require('loopback-datasource-juggler/test/manipulation.test.js');
```